apps/astarte_realm_management_api/test/astarte_realm_management_api_web/controllers/version_controller_test.exs

```diff
@@ -0,0 +1,53 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.RealmManagement.APIWeb.VersionControllerTest do
+  use Astarte.RealmManagement.APIWeb.ConnCase
+
+  alias Astarte.RealmManagement.API.Helpers.JWTTestHelper
+  alias Astarte.RealmManagement.API.Helpers.RPCMock.DB
+
+  @realm "testrealm"
```

Comment:
In general, it would be great to allow tests to run concurrently (`async: true`) and, when doing that, hardcoding the same realm in all tests might lead to some race conditions. It is not compulsory to do so in this PR!
As an example, you could check https://github.com/astarte-platform/astarte/pull/1211